docs/tests.md

```diff
@@ -12,7 +12,12 @@ found in the [general test directory](/test/general).
 
 In order to run tests from your world, you will need to create a `test` package within your world package. This can be
 done by creating a `test` directory with a file named `__init__.py` inside it inside your world. By convention, a base
-for your world tests can be created in this file that you can then import into other modules.
+for your world tests can be created in `bases.py` or any file that does not start with `test`, that you can then import
```

Comment:
didn't we specifically go with `bases` because the old base file was `Base`? i think it'd be better to have `base.py` as convention but that's splitting hairs so 🤷

Reply:
Yes, we picked bases.py because git on windows won't rename Base.py to base.py, however I think following the core convention is better than having a slightly better name.